composer.json

```diff
@@ -28,7 +28,7 @@
         "phpunit/php-code-coverage": "@stable",
         "symfony/form": "~2.3 || ~3.0",
         "doctrine/phpcr-odm": "~1.3@dev",
-        "symfony-cmf/testing": "~1.2,>=1.2.7",
+        "symfony-cmf/testing": "dev-support-symfony-3 as 1.2.8",
```

Comment:
if this works, we need to merge https://github.com/symfony-cmf/Testing/pull/113, tag Testing and adjust this line.